deploy/runtime/cashflow.lk

```diff
@@ -1308,12 +1318,15 @@ function set_om(xl) {
 
 	if ( is_fom_fin )
```

Comment:
Is this trying to use the is_fom_fin function? If so parenthesis are missing.

Reply:
Ooops!